datasets/bbc_hindi_nli/README.md

```diff
@@ -0,0 +1,181 @@
+
+### Source Data
+
+Source Dataset for the recasting process is the BBC Hindi Headlines Dataset(https://github.com/NirantK/hindi2vec/releases/tag/bbc-hindi-v0.1)
+
+#### Initial Data Collection and Normalization
+
+-  BBC Hindi News Classification Dataset contains 4, 335 Hindi news headlines tagged across 14 categories: India, Pakistan,news, International, entertainment, sport, science, China, learning english, social, southasia, business, institutional, multimedia
+-  We processed this dataset to combine two sets of relevant but low prevalence classes.
+- Namely, we merged the samples from Pakistan, China, international, and southasia as one class called international.
+- Likewise, we also merged samples from news, business, social, learning english, and institutional as news.
+- Lastly, we also removed the class multimedia because there were very few samples.
+
+#### Who are the source language producers?
+
+Pls refer to this paper: "https://www.aclweb.org/anthology/2020.aacl-main.71"
+
+### Annotations
+
+#### Annotation process
+
+Annotation process has been described in Dataset Creation Section.
+
+#### Who are the annotators?
+
+Annotation is done automatically.
+
+### Personal and Sensitive Information
+
+No Personal and Sensitive Information is mentioned in the Datasets.
+
+## Considerations for Using the Data
+
+Pls refer to this paper: https://www.aclweb.org/anthology/2020.aacl-main.71
+
+### Discussion of Biases
+
+Pls refer to this paper: https://www.aclweb.org/anthology/2020.aacl-main.71
+
+### Other Known Limitations
+
+No other known limitations
+
+## Additional Information
+
+Pls refer to this link: https://github.com/midas-research/hindi-nli-data
+
+### Dataset Curators
+
+- This corpus can be used freely for research purposes.
+- The paper listed below provide details of the creation and use of the corpus. If you use the corpus, then please cite the paper.
+- If interested in commercial use of the corpus, send email to midas@iiitd.ac.in.
+- If you use the corpus in a product or application, then please credit the authors and Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi appropriately. Also, if you send us an email, we will be thrilled to know about how you have used the corpus.
+- Multimodal Digital Media Analysis Lab - Indraprastha Institute of Information Technology, New Delhi, India disclaims any responsibility for the use of the corpus and does not provide technical support. However, the contact listed above will be happy to respond to queries and clarifications.
+- Rather than redistributing the corpus, please direct interested parties to this page
+- Please feel free to send us an email:
+  - with feedback regarding the corpus.
+  - with information on how you have used the corpus.
+  - if interested in having us analyze your data for natural language inference.
+  - if interested in a collaborative research project.
```

Comment:
Is this a quote from the paper ? If so can you make it explicit by adding something like `It is written in the paper (link to the paper) that:` before the paragraph

Reply:
It was actually taken from a repo. I have added the repo link above the para.